src/main/java/de/uhd/ifi/se/decision/management/jira/rest/KnowledgeRest.java

```diff
@@ -151,6 +151,10 @@ public Response createDecisionKnowledgeElement(@Context HttpServletRequest reque
 		existingElement.setDocumentationLocation(documentationLocationOfExistingElement);
 		existingElement.setProject(element.getProject().getProjectKey());
 
+		AbstractPersistenceManagerForSingleLocation persistenceManagerForExistingElement =
```

Comment:
The "existingElement" is overwritten like this. Please change it to:
`AbstractPersistenceManagerForSingleLocation persistenceManagerForExistingElement =	PersistenceManager.getOrCreate(element.getProject().getProjectKey()).getPersistenceManager(docuLocu)
`